doc/api_review/core/keyNew.md

```diff
@@ -85,13 +85,14 @@
 - [x] Function code is fully covered by tests
 - [x] All possible error states are covered by tests
 - [ ] All possible enum values are covered by tests  
-       - [ ] KEY_META  
-       - [ ] KEY_FLAGS
+       - [x] KEY_META  
+       - [x] KEY_FLAGS
 - [ ] No inconsistencies between tests and documentation  
        - [ ] Documentation says i can work with `Key *k =keyNew("", KEY_END);`  
```

Comment:
What would be the proper behaviour here?

Reply:
Can you elaborate please?

Reply:
Do you have the question as answered here? https://github.com/ElektraInitiative/libelektra/pull/3825#issuecomment-840856334

So empty key names are not possible anymore: now we have an invariant that the key name is always valid. The above statement would fail (k ==null). The leftovers need to be cleaned up and the invariant better documented.

Reply:
Okay, so the documentation would need to be changed then.

Reply:
should be fixed now in the documentation

Reply:
I do not think that `keyNew("", KEY_END);` should work. It is best to remove that legacy functionality and remove the "Alternative" example in examples/keyNew.c